examples/helloworld-cdi2-se/pom.xml

```diff
@@ -81,6 +80,23 @@
                 </plugins>
             </build>
         </profile>
+        <!--<profile>
```

Comment:
Do we need this when it's commented out?

Reply:
fixed